library/styleguideComponents/ComponentsList/index.js

```diff
@@ -84,6 +88,21 @@ export default class ComponentsList extends Component {
                         label="Designs"
                         href="/#/Designs"
                     />
+                    <StyledButton onClick={this.handleOnClickCustomWorkBtn} ref={this.itemRef}>
+                        <Icon className="rainbow-m-bottom_x-small" as={CustomWorkIcon} />
+                        Custom Work
+                    </StyledButton>
+                    <InternalOverlay
+                        isVisible={isVisible}
+                        render={() => {
+                            return (
+                                <ContactUsForm
+                                    onSuccessMessageSent={this.handleOnClickCustomWorkBtn}
+                                />
+                            );
+                        }}
```

Comment:
here now you can pass ContactUsForm as children which is better since render function is declared inline and create a new component in each rerender